charts/jenkins/templates/config.yaml

```diff
@@ -209,7 +209,12 @@ data:
     # Install missing plugins
     cp /var/jenkins_config/plugins.txt {{ .Values.master.jenkinsHome }};
     rm -rf {{ .Values.master.jenkinsRef }}/plugins/*.lock
-    /usr/local/bin/install-plugins.sh `echo $(cat {{ .Values.master.jenkinsHome }}/plugins.txt)`;
+    version () { echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }'; }
+    if [ -n "$(command -v jenkins-plugin-cli)" 2>/dev/null ] && [ $(version $(jenkins-plugin-cli --version)) -ge $(version "2.1.1") ]; then
+      jenkins-plugin-cli --war {{ .Values.master.jenkinsRef }}/jenkins.war --plugin-file {{ .Values.master.jenkinsHome }}/plugins.txt;
```

Comment:
Do we need to check if `{{ .Values.master.jenkinsRef }}/jenkins.war` exists?

Reply:
i don't know, do we? doesn't most of the chart assume ref has content?

Reply:
Not sure myself. It's possible that people use their own image. Need to check other references to see if this is a problem.

Reply:
I have the plugins installed directly into my image, so I personally disable the install plugins feature (I like being able to roll back and not need the internet to startup)

lemme know if you want me to add the if check

Reply:
@halkeye can you ping me once you added the check?

Reply:
@torstenwalter totally got distracted and didn't get to it.

Added a check to use the home dir war file if the ref doesn't exist